image/templates/helm/shared/config-templates/scanner-v4/matcher-config.yaml.tpl

```diff
@@ -0,0 +1,27 @@
+{{- /*
+  This is the configuration file template for the Scanner v4 Matcher.
+  Except for in extremely rare circumstances, you DO NOT need to modify this file.
+  All config options that are possibly dynamic are templated out and can be modified
+  via `--set`/values-files specified via `-f`.
+     */ -}}
+
+# Configuration file for Scanner v4 Matcher.
+
+indexer:
```

Comment:
this is also not set in stone. We may potentially even want to consider just using a single configuration which has configs for both the indexer and matcher, if we want to consume fewer k8s resources